web/src/message_viewport.ts

```diff
@@ -23,13 +23,6 @@ const cached_height = new util.CachedValue({compute_value: () => $scroll_contain
 export const width = cached_width.get.bind(cached_width);
 export const height = cached_height.get.bind(cached_height);
 
-// TODO: This function lets us use the DOM API instead of jquery
-// (<10x faster) for condense.js, but we want to eventually do a
-// bigger of refactor `height` and `width` above to do the same.
-export function max_message_height(): number {
-    return document.querySelector("html")!.offsetHeight * 0.65;
-}
```

Comment:
Hmm, this TODO seems important; it sounds like `$scroll_container.width()` maybe should be using `document.querySelector` as a prep refactor?

Worth profiling to see if it matters, but I seem to recall computing heights for condensing was a significant cost in rendering message feeds. I guess we only call this once, though, so maybe doesn't really matter? Just seems worth thinking about.

Reply:
87649ddbab66fdd7117c9bf99282cc9555031fa6 is the context.

Reply:
Yeah, since we are using `util.CachedValue` now, this no longer is performance issue as we are only computing the height once.